### DISCUSSION TITLE
Não consegui rodar o Update Database do workflow do curso de Azure, Git, Github e DevOps

### DISCUSSION THREAD

**Discussion by oGrandeK** [balta-io/forum#295] [Tira Dúvidas] [answered]
Olá novamente professor, estava seguindo seus passos do curso 2812 e quando devemos rodar o workflow minha action dá erro, segue abaixo o erro e meu workflow: 
![image](https://user-images.githubusercontent.com/89154726/228996649-32278282-284d-4ae0-9e70-15933bcba720.png)

![image](https://user-images.githubusercontent.com/89154726/228996703-959676ea-fe38-4630-9525-ee41cef69799.png)

Meu Docker está rodando e está tudo funcionando, o que devo fazer?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @oGrandeK você está tentando rodar local? Acredito que no curso mostramos no Azure com SQL Azure... correto?

**Reply by oGrandeK** [reply]
Boa noite novamente, foi resolvido sabe como? bastei olhar as próximas aulas que o senhor tinha feito que já tinha a solução kkkkk.
Tive que adicionar um passos a mais no workflow pra logar no Azure, pegar IP, e adicionar o ip no servidor SQL.

Uma dúvida que me surgiu, toda vez que for subir algo no Azure vou ter que decorar esse workflow que o senhor atualizou pra poder subir? Tem algum lugar onde eu possa achar essas dicas na hora de subir pro GitHub Actions? Ou o senhor acha melhor eu salvar num TXT mesmo kkkkk?

**Reply by andrebaltieri** [reply]
Show 🚀
